### PR TITLE
Disabling broken tests for now

### LINKED ISSUE #27524: [Build] Darwin python parser no-ble-tsan failing
### Build issue(s)

https://github.com/project-chip/connectedhomeip/actions/runs/5398386504/jobs/9804137158?pr=27516

```
ERROR   08:39:54.460 - TEST OUT  : 		    [IN] (U) Sending msg 6874998 to IP address 'UDP:[fd00:0:1:1::3]:5540'
ERROR   08:39:54.460 - TEST OUT  : 		    [SC] Sent Sigma1 msg
ERROR   08:39:54.460 - TEST OUT  : 		    [DIS] OperationalSessionSetup[1:0000000012344321]: State change 3 --> 4
ERROR   08:39:54.460 - TEST OUT  : 		    [DIS] Failed to parse mDNS query
ERROR   08:39:54.460 - TEST OUT  : 		    [EM] Retransmitting MessageCounter:6874998 on exchange 3292i Send Cnt 1
ERROR   08:39:54.460 - TEST OUT  : 		    [IN] (U) Sending msg 6874998 to IP address 'UDP:[fd00:0:1:1::3]:5540'
ERROR   08:39:54.460 - TEST OUT  : 		    [DIS] Failed to parse mDNS query
ERROR   08:39:54.460 - TEST OUT  : 		    [DIS] Failed to parse mDNS query
ERROR   08:39:54.460 - TEST OUT  : 		    [EM] Retransmitting MessageCounter:6874998 on exchange 3292i Send Cnt 2
ERROR   08:39:54.460 - TEST OUT  : 		    [IN] (U) Sending msg 6874998 to IP address 'UDP:[fd00:0:1:1::3]:5540'
ERROR   08:39:54.460 - TEST OUT  : 		    [DIS] Failed to parse mDNS query
ERROR   08:39:54.460 - TEST OUT  : 		    [DIS] Failed to parse mDNS query
ERROR   08:39:54.460 - TEST OUT  : 		    [EM] Retransmitting MessageCounter:6874998 on exchange 3292i Send Cnt 3
ERROR   08:39:54.461 - TEST OUT  : 		    [IN] (U) Sending msg 6874998 to IP address 'UDP:[fd00:0:1:1::3]:5540'
ERROR   08:39:54.461 - TEST OUT  : 		    [DIS] Failed to parse mDNS query
ERROR   08:39:54.461 - TEST OUT  : 		    [DIS] Failed to parse mDNS query
ERROR   08:39:54.461 - TEST OUT  : 		    [DIS] Failed to parse mDNS query
ERROR   08:39:54.461 - TEST OUT  : 		    [EM] Retransmitting MessageCounter:6874998 on exchange 3292i Send Cnt 4
ERROR   08:39:54.461 - TEST OUT  : 		    [IN] (U) Sending msg 6874998 to IP address 'UDP:[fd00:0:1:1::3]:5540'
ERROR   08:39:54.461 - TEST OUT  : 		    [DIS] Failed to parse mDNS query
ERROR   08:39:54.461 - TEST OUT  : 		    [EM] Failed to Send CHIP MessageCounter:6874998 on exchange 3292i sendCount: 4 max retries: 4
ERROR   08:39:54.461 - TEST OUT  : 		    [SC] CASESession timed out while waiting for a response from the peer. Current state was 1
ERROR   08:39:54.461 - TEST OUT  : 		    [IN] SecureSession[0x7b3c000078f0]: Released - Type:2 LSID:63073
ERROR   08:39:54.461 - TEST OUT  : 		    [DIS] OperationalSessionSetup[1:0000000012344321]: State change 4 --> 2
ERROR   08:39:54.461 - TEST OUT  : 		    [DIS] OperationalSessionSetup[1:0000000012344321]: State change 2 --> 1
ERROR   08:39:54.462 - TEST OUT  : 		    [-] src/protocols/secure_channel/CASESession.cpp:493: CHIP Error 0x00000032: Timeout at ../../examples/chip-tool/commands/delay/WaitForCommissioneeCommand.cpp:46
ERROR   08:39:54.462 - TEST OUT  : 		    [TOO] Run command failure: src/protocols/secure_channel/CASESession.cpp:493: CHIP Error 0x00000032: Timeout
ERROR   08:39:54.462 - TEST OUT  : 		    Expected: [
ERROR   08:39:54.462 - TEST OUT  : 		                {
ERROR   08:39:54.462 - TEST OUT  : 		                  "values": [
ERROR   08:39:54.462 - TEST OUT  : 		                    {}
ERROR   08:39:54.462 - TEST OUT  : 		                  ]
ERROR   08:39:54.462 - TEST OUT  : 		                }
ERROR   08:39:54.462 - TEST OUT  : 		              ]
ERROR   08:39:54.462 - TEST OUT  : 		    Received: [
ERROR   08:39:54.462 - TEST OUT  : 		                 {
ERROR   08:39:54.462 - TEST OUT  : 		                   "error": "FAILURE"
ERROR   08:39:54.462 - TEST OUT  : 		                 }
ERROR   08:39:54.462 - TEST OUT  : 		               ]
ERROR   08:39:54.462 - TEST OUT  : 		***** Test Failure : The test expects no error but the "FAILURE" error occured.
ERROR   08:39:54.462 - TEST OUT  : 		**** Test Setup: Device Connection Failure [deviceId=305414945. Error The test expects no error but the "FAILURE" error occured.]
ERROR   08:39:54.463 - TEST OUT  : ✗ Test finished in 6461ms with 0 success, 1 errors and 0 warnings
ERROR   ================ CAPTURED LOG END ====================
ERROR   TestSelfFabricRemoval          - FAILED in 9.78 seconds
Traceback (most recent call last):
  File "/__w/connectedhomeip/connectedhomeip/./scripts/tests/run_test_suite.py", line 344, in cmd_run
    test.Run(
  File "/__w/connectedhomeip/connectedhomeip/scripts/tests/chiptest/test_definition.py", line 340, in Run
    runner.RunSubprocess(
  File "/__w/connectedhomeip/connectedhomeip/scripts/tests/chiptest/runner.py", line 167, in RunSubprocess
    raise Exception('Command %r failed: %d' % (cmd, s.returncode))
Exception: Command ['ip', 'netns', 'exec', 'tool', 'python3', '/__w/connectedhomeip/connectedhomeip/scripts/tests/yaml/chiptool.py', 'tests', 'TestSelfFabricRemoval', '--PICS', 'src/app/tests/suites/certification/ci-pics-values', '--server_path', './out/linux-x64-chip-tool-no-ble-tsan-clang/chip-tool', '--server_arguments', 'interactive server --storage-directory /tmp/tmphgafbthn'] failed: 1
Error: Process completed with exit code 2.
```

### Platform

_No response_

### Anything else?

_No response_

### LINKED ISSUE #27673: [Build] Disabled "Run Tests using the python parser sending commands to chip-tool"
### Build issue(s)

Due to: https://github.com/project-chip/connectedhomeip/issues/27524

I disabled this test step.

### Platform

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by woody-apple**
Fixes https://github.com/project-chip/connectedhomeip/issues/27524

Disabling these for now

Filed this as a follow up: https://github.com/project-chip/connectedhomeip/issues/27673

### CODE REVIEW

**Review by jmartinez-silabs — APPROVED** [review]

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -349,23 +350,24 @@ jobs:
...
-                     --all-clusters-app ./out/darwin-x64-all-clusters-${BUILD_VARIANT}/chip-all-clusters-app \
-                     --lock-app ./out/darwin-x64-lock-${BUILD_VARIANT}/chip-lock-app \
-                     --ota-provider-app ./out/darwin-x64-ota-provider-${BUILD_VARIANT}/chip-ota-provider-app \
-                     --ota-requestor-app ./out/darwin-x64-ota-requestor-${BUILD_VARIANT}/chip-ota-requestor-app \
-                     --tv-app ./out/darwin-x64-tv-app-${BUILD_VARIANT}/chip-tv-app \
-                     --bridge-app ./out/darwin-x64-bridge-${BUILD_VARIANT}/chip-bridge-app \
-                  "
+            #   https://github.com/project-chip/connectedhomeip/issues/27673
+            # - name: Run Tests using the python parser sending commands to chip-tool
+            #   run: |
```
Comment (bzbarsky-apple) [inline .github/workflows/tests.yaml:355]: If we just want to disable this for the tsan variant (which is what seems to be failing), how about:

1. We move the `if: matrix.build_variant != 'no-ble-tsan-clang'` to this job.
2. We remove it from the "Run Tests" immediately above.

Alternately, `if: false` or something might be better in terms of disabling, but then we should still remove the bit from the job above, or disable the tsan variant altogether, so we don't waste time on it if it's not running any tests.

File: .github/workflows/tests.yaml [hunk 2]
```diff
@@ -211,22 +211,23 @@ jobs:
...
-                     --test-timeout-seconds 120 \
-                     --all-clusters-app ./out/linux-x64-all-clusters-${BUILD_VARIANT}/chip-all-clusters-app \
-                     --lock-app ./out/linux-x64-lock-${BUILD_VARIANT}/chip-lock-app \
-                     --ota-provider-app ./out/linux-x64-ota-provider-${BUILD_VARIANT}/chip-ota-provider-app \
-                     --ota-requestor-app ./out/linux-x64-ota-requestor-${BUILD_VARIANT}/chip-ota-requestor-app \
-                     --tv-app ./out/linux-x64-tv-app-${BUILD_VARIANT}/chip-tv-app \
-                     --bridge-app ./out/linux-x64-bridge-${BUILD_VARIANT}/chip-bridge-app \
-                  "
+            #   https://github.com/project-chip/connectedhomeip/issues/27673
+            # - name: Run Tests using the python parser sending commands to chip-tool
```
Comment (bzbarsky-apple) [inline .github/workflows/tests.yaml:215]: Why are we disabling the Linux job, if the failures we are fixing are on Darwin?

Comment (woody-apple) [reply .github/workflows/tests.yaml:215]: https://github.com/project-chip/connectedhomeip/issues/27524

File: .github/workflows/tests.yaml (hunk 1 above)
Comment (woody-apple) [reply .github/workflows/tests.yaml]: Done

File: .github/workflows/tests.yaml (hunk 2 above)
Comment (bzbarsky-apple) [reply .github/workflows/tests.yaml]: OK, or https://github.com/project-chip/connectedhomeip/actions/runs/5479204916/jobs/9980729852?pr=27669 which is also a Linux fail....

**Review by bzbarsky-apple — APPROVED** [review]

**Pull request merged**